pandas/core/tools/timedeltas.py

```diff
@@ -174,12 +177,14 @@ def to_timedelta(
     TimedeltaIndex(['0 days', '1 days', '2 days', '3 days', '4 days'],
                    dtype='timedelta64[ns]', freq=None)
     """
-    if unit in {"T", "t", "L", "l"}:
+    if unit in DEPR_ABBREVS:
         warnings.warn(
-            f"Unit '{unit}' is deprecated and will be removed in a future version.",
+            f"Unit '{unit}' is deprecated and will be removed in a future version. "
+            f"Please use '{DEPR_ABBREVS.get(unit)}' instead of '{unit}'.",
             FutureWarning,
             stacklevel=find_stack_level(),
         )
+        unit = DEPR_ABBREVS.get(unit)
 
```

Comment:
I think you can just remove all of this now, as `parse_timedelta_unit` should raise the warning?

Reply:
Thank you, I agree. It was my thought too. I removed `warnings.warn` here.